Makefile

```diff
@@ -489,6 +498,7 @@ disk_prepare: build $(SOTNDISK)
 	cp $(BUILD_DIR)/MAR.BIN $(DISK_DIR)/BOSS/MAR/MAR.BIN
 	cp $(BUILD_DIR)/F_MAR.BIN $(DISK_DIR)/BOSS/MAR/F_MAR.BIN
 	cp $(BUILD_DIR)/TT_000.BIN $(DISK_DIR)/SERVANT/TT_000.BIN
+	cp $(BUILD_DIR)/TT_001.BIN $(DISK_DIR)/SERVANT/TT_001.BIN
 disk: disk_prepare
 	$(SOTNDISK) make build/sotn.$(VERSION).cue $(DISK_DIR) $(CONFIG_DIR)/disk.us.lba
 disk_debug: disk_prepare
```

Comment:
`disk_debug` does something with `TT_000.BIN` which I was unclear on, do I need to add anything in there too for `TT_001.BIN`?

Reply:
We have the option of replacing the bat overlay with a debugging overlay instead. I don't think you need to make changes for it https://github.com/Xeeynamo/sotn-decomp/tree/master/tools/sotn-debugmodule